variables.tf

```diff
@@ -22,21 +22,25 @@
 # Configure the IBM Cloud provider
 ################################################################
 variable "ibmcloud_api_key" {
+    type        = string
```

Comment:
Need validation as this is a required variable.

Reply:
Added validation to check empty string and default value.